analysis/wikitrust_algorithms/text_trust/block.py

```diff
@@ -0,0 +1,47 @@
+"""
+Module containing Block class. Part of the text_trust package.
+
+Eric Vin, 2019-2020
+"""
+
+#Standard library imports
+from typing import List
+
+#text_trust package imports
+from .word import Word
+
+#text_diff package imports
+from wikitrust_algorithms.text_diff.edit import Edit
+
+class Block:
+    """
+    Class representing a group of words that are all part of one edit.
+    """
+
```

Comment:
I am wondering whether this great object oriented organization, beyond some point, just makes the code very heavy.  What are you planning to do with a block here?  Let's discuss.

Reply:
I use the block class to encapsulate the boundaries of each type of edit. For me personally, I think it adds a lot of clarity to the edge effect calculations, among other uses. I have yet to write comprehensive documentation for all the algorithms at a high level, so hopefully that will clarify things. I am also of course, removing the class in it's entirety if you think that's the best option.